itest/rust/build.rs

```diff
@@ -280,6 +338,164 @@ fn generate_rust_methods(inputs: &[Input]) -> Vec<TokenStream> {
+            #[export(flags_3d_render)]
+            export_flags_3d_render: i64,
+            #[export(flags_3d_navigation)]
+            export_flags_3d_navigation: i64,
+            #[export(enum = (Warrior, Magician, Thief))]
+            export_enum_int_warrior_magician_thief: i64,
+            #[export(enum = (Slow = 30, Average = 60, VeryFast = 200))]
+            export_enum_int_slow_30_average_60_very_fast_200: i64,
+            #[export(enum = (Rebecca, Mary, Leah))]
+            export_enum_string_rebecca_mary_leah: GodotString,
+        }
+
+        #[godot_api]
+        impl PropertyTemplateRust {}
+    };
+
+    gdscript.extend([
+        r#"@export_file var export_file: String"#.to_string(),
+        r#"@export_file("*.txt") var export_file_wildcard_txt: String"#.to_string(),
+        r#"@export_global_file var export_global_file: String"#.to_string(),
+        r#"@export_global_file("*.png") var export_global_file_wildcard_png: String"#.to_string(),
+        r#"@export_dir var export_dir: String"#.to_string(),
+        r#"@export_global_dir var export_global_dir: String"#.to_string(),
+        r#"@export_multiline var export_multiline: String"#.to_string(),
+        r#"@export_range(0, 20) var export_range_float_0_20: float"#.to_string(),
+        r#"@export_range(-10, 20, 0.2) var export_range_float_neg_10_20_02: float"#.to_string(),
+        r#"@export_range(0, 100, 1, "or_greater", "or_less") var export_range_int_0_100_1_or_greater_or_less: int"#.to_string(),
+        r#"@export_exp_easing var export_exp_easing: float"#.to_string(),
+        r#"@export_color_no_alpha var export_color_no_alpha: Color"#.to_string(),
+        r#"@export_node_path("Button", "TouchScreenButton") var export_node_path_button_touch_screen_button: NodePath"#.to_string(),
+        r#"@export_flags("Fire", "Water", "Earth", "Wind") var export_flags_fire_water_earth_wind: int"#.to_string(),
+        r#"@export_flags("Self:4", "Allies:8", "Foes:16") var export_flags_self_4_allies_8_foes_16: int"#.to_string(),
+        r#"@export_flags_2d_physics var export_flags_2d_physics: int"#.to_string(),
+        r#"@export_flags_2d_render var export_flags_2d_render: int"#.to_string(),
+        r#"@export_flags_2d_navigation var export_flags_2d_navigation: int"#.to_string(),
+        r#"@export_flags_3d_physics var export_flags_3d_physics: int"#.to_string(),
+        r#"@export_flags_3d_render var export_flags_3d_render: int"#.to_string(),
+        r#"@export_flags_3d_navigation var export_flags_3d_navigation: int"#.to_string(),
+        r#"@export_enum("Warrior", "Magician", "Thief") var export_enum_int_warrior_magician_thief: int"#.to_string(),
+        r#"@export_enum("Slow:30", "Average:60", "VeryFast:200") var export_enum_int_slow_30_average_60_very_fast_200: int"#.to_string(),
+        r#"@export_enum("Rebecca", "Mary", "Leah") var export_enum_string_rebecca_mary_leah: String"#.to_string(),
+    ]);
+
+    let gdscript = gdscript.join("\n");
```

Comment:
Could you maybe use one big multi-line string here and avoid the `join()`?
Would also have less noise around `r# ... #.to_string()`